packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.service.js

```diff
@@ -20,4 +20,8 @@ export default class OverTheBoxDetailsService {
       })
       .catch(() => []);
   }
+
+  getDeviceHardware(serviceName) {
+    return this.$http.get(`/overTheBox/${serviceName}/device/hardware`);
```

Comment:
To avoid doing it for each call
```suggestion
    return this.$http.get(`/overTheBox/${serviceName}/device/hardware`).then(( {data} ) => data) ;
```